<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindow</class>
 <widget class="QDialog" name="mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>瓦片切分工具</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_9">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>1、选择并打开数据源</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;请选择数据类型：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <property name="spacing">
                 <number>20</number>
                </property>
                <item>
                 <widget class="QRadioButton" name="FileType1">
                  <property name="text">
                   <string>影像</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                  <attribute name="buttonGroup">
                   <string notr="true">FileTypeGroup</string>
                  </attribute>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="FileType2">
                  <property name="text">
                   <string>DEM</string>
                  </property>
                  <attribute name="buttonGroup">
                   <string notr="true">FileTypeGroup</string>
                  </attribute>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>32</width>
                  <height>17</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;打开文件路径：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="FilePathshow"/>
              </item>
              <item>
               <widget class="QPushButton" name="OpenFile">
                <property name="focusPolicy">
                 <enum>Qt::StrongFocus</enum>
                </property>
                <property name="text">
                 <string>打开文件</string>
                </property>
                <property name="autoDefault">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
         <zorder></zorder>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>2、数据源信息属性</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QFormLayout" name="formLayout_2">
              <property name="fieldGrowthPolicy">
               <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;空间经纬度范围：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <layout class="QFormLayout" name="formLayout">
                <property name="fieldGrowthPolicy">
                 <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
                </property>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;上：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLineEdit" name="Top">
                  <property name="minimumSize">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;下：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLineEdit" name="Bottom">
                  <property name="minimumSize">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;左：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QLineEdit" name="Left">
                  <property name="minimumSize">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QLineEdit" name="Right">
                  <property name="minimumSize">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;右：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0">
               <layout class="QFormLayout" name="formLayout_3">
                <item row="0" column="0">
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;瓦片金字塔层次：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <layout class="QVBoxLayout" name="verticalLayout_2">
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_6">
                    <item>
                     <widget class="QLabel" name="label_9">
                      <property name="text">
                       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;最小层：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="txtMinLevel"/>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_7">
                    <item>
                     <widget class="QLabel" name="label_10">
                      <property name="text">
                       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;最大层：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="txtMaxLevel"/>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item row="0" column="1">
               <widget class="QPushButton" name="Calculation">
                <property name="focusPolicy">
                 <enum>Qt::StrongFocus</enum>
                </property>
                <property name="text">
                 <string>计算</string>
                </property>
                <property name="autoDefault">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
         <zorder></zorder>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>3、输入输出信息</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;保存文件路径：&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="SavePathShow"/>
            </item>
            <item>
             <widget class="QPushButton" name="SaveFile">
              <property name="focusPolicy">
               <enum>Qt::StrongFocus</enum>
              </property>
              <property name="text">
               <string>保存文件</string>
              </property>
              <property name="autoDefault">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
         <zorder></zorder>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="structure">
           <property name="focusPolicy">
            <enum>Qt::StrongFocus</enum>
           </property>
           <property name="text">
            <string>构建</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="close">
           <property name="focusPolicy">
            <enum>Qt::StrongFocus</enum>
           </property>
           <property name="text">
            <string>关闭</string>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2</width>
     <height>2</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_4"/>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2</width>
     <height>2</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3"/>
  </widget>
 </widget>
 <tabstops>
  <tabstop>FileType1</tabstop>
  <tabstop>FileType2</tabstop>
  <tabstop>FilePathshow</tabstop>
  <tabstop>OpenFile</tabstop>
  <tabstop>Top</tabstop>
  <tabstop>Bottom</tabstop>
  <tabstop>Left</tabstop>
  <tabstop>Right</tabstop>
  <tabstop>txtMinLevel</tabstop>
  <tabstop>txtMaxLevel</tabstop>
  <tabstop>Calculation</tabstop>
  <tabstop>SavePathShow</tabstop>
  <tabstop>SaveFile</tabstop>
  <tabstop>structure</tabstop>
  <tabstop>close</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="FileTypeGroup"/>
 </buttongroups>
</ui>
